install error

osx

```
(reactopya) Bens-MBP-2:reactopya_gallery_jupyter bendichter$ yarn install
yarn install v1.17.3
info No lockfile found.
[1/4] 🔍  Resolving packages...
warning plotly.js > regl-splom > left-pad@1.3.0: use String.prototype.padStart()
warning plotly.js > ndarray-fill > cwise > static-module > through2 > xtend > object-keys@0.4.0: 
warning plotly.js > point-cluster > bubleify > buble > os-homedir@2.0.0: This is not needed anymore. Use `require('os').homedir()` instead.
warning @tulipjs/eslint-config > eslint > file-entry-cache > flat-cache > circular-json@0.3.3: CircularJSON is in maintenance only, flatted is its successor.
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
success Saved lockfile.
$ webpack
Hash: 565832ae7dfe3648e9b03d446ff33bd744396e854cde152d81600f104c91
Version: webpack 4.39.1
Child
    Hash: 565832ae7dfe3648e9b0
    Time: 959ms
    Built at: 08/06/2019 12:48:23 PM
           Asset      Size  Chunks             Chunk Names
    extension.js  1.25 KiB       0  [emitted]  main
    Entrypoint main = extension.js
    [0] multi ./lib/extension.js 28 bytes {0} [built]
    [1] ./lib/extension.js 860 bytes {0} [built]
    
    WARNING in configuration
    The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
    You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
Child
    Hash: 3d446ff33bd744396e85
    Time: 4195ms
    Built at: 08/06/2019 12:48:26 PM
     2 assets
    Entrypoint main = index.js index.js.map
     [3] (webpack)/buildin/global.js 472 bytes {0} [built]
     [4] multi ./lib/index.js 28 bytes {0} [built]
     [5] ./lib/index.js 180 bytes {0} [built]
     [6] ./lib/all_widgets.js 4.74 KiB {0} [built]
    [11] external "@jupyter-widgets/base" 42 bytes {0} [built]
    [13] (webpack)/buildin/module.js 497 bytes {0} [built]
    [14] ./package.json 1.45 KiB {0} [built]
        + 8 hidden modules
    
    WARNING in configuration
    The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
    You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/Autocorrelograms/Autocorrelograms' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 10:0-119 78:40-56
     @ ./lib/index.js
     @ multi ./lib/index.js
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/ElectrodeGeometry/ElectrodeGeometry' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 12:0-122 141:40-57
     @ ./lib/index.js
     @ multi ./lib/index.js
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/InteractivePlotlyExample/InteractivePlotlyExample' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 9:0-143 44:40-64
     @ ./lib/index.js
     @ multi ./lib/index.js
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/PlotlyExample/PlotlyExample' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 11:0-110 107:40-53
     @ ./lib/index.js
     @ multi ./lib/index.js
Child
    Hash: 4cde152d81600f104c91
    Time: 4493ms
    Built at: 08/06/2019 12:48:27 PM
     2 assets
    Entrypoint main = index.js index.js.map
     [3] (webpack)/buildin/global.js 472 bytes {0} [built]
     [4] multi ./lib/embed.js 28 bytes {0} [built]
     [5] ./lib/embed.js 454 bytes {0} [built]
     [6] ./lib/all_widgets.js 4.74 KiB {0} [built]
    [11] external "@jupyter-widgets/base" 42 bytes {0} [built]
    [13] (webpack)/buildin/module.js 497 bytes {0} [built]
    [14] ./package.json 1.45 KiB {0} [built]
        + 8 hidden modules
    
    WARNING in configuration
    The 'mode' option has not been set, webpack will fallback to 'production' for this value. Set 'mode' option to 'development' or 'production' to enable defaults for each environment.
    You can also set it to 'none' to disable any default behavior. Learn more: https://webpack.js.org/configuration/mode/
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/Autocorrelograms/Autocorrelograms' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 10:0-119 78:40-56
     @ ./lib/embed.js
     @ multi ./lib/embed.js
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/ElectrodeGeometry/ElectrodeGeometry' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 12:0-122 141:40-57
     @ ./lib/embed.js
     @ multi ./lib/embed.js
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/InteractivePlotlyExample/InteractivePlotlyExample' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 9:0-143 44:40-64
     @ ./lib/embed.js
     @ multi ./lib/embed.js
    
    ERROR in ./lib/all_widgets.js
    Module not found: Error: Can't resolve '../reactopya_gallery/reactopya_gallery/PlotlyExample/PlotlyExample' in '/Users/bendichter/dev/reactopya_gallery_jupyter/lib'
     @ ./lib/all_widgets.js 11:0-110 107:40-53
     @ ./lib/embed.js
     @ multi ./lib/embed.js
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

--- a/setup.py
+++ b/setup.py
@@ -10,7 +10,7 @@ setuptools.setup(
     description="",
     packages=setuptools.find_packages(),
     scripts=[],
-    install_requires=[],
+    install_requires=['jupyter'],
     classifiers=(
         "Programming Language :: Python :: 3",
         "License :: OSI Approved :: Apache Software License",

